gateway/config/development.lua

```diff
@@ -28,5 +28,6 @@ return {
     lua_code_cache = 'on',
     configuration_loader = 'lazy',
     configuration_cache = 0,
-    configuration = data_url('application/json', configuration)
+    configuration = data_url('application/json', configuration),
+    port = { metrics = 9100 },
```

Comment:
@mikz maybe reserve a port for the APIcast endpoint so we can publish it on the [Prometheus Default Port Allocation list](https://github.com/prometheus/prometheus/wiki/Default-port-allocations)?

Reply:
I think we should agree how we want to proceed first.
We are not going to reserve a port for every app we have, right? We need some generic port for openresty exporter?

Reply:
Got It, I thought that at least for APICast we were willing to create a specific "exporter" but yes makes more sense to create a generic port for openresty exporter.

Reply:
To me it doesn't really make sense to reserve a port for each APP, but this is not a /generic/ openresty prometheus exporter, so, should we define a port for the "APIcast framework" and reuse that one in all our applications?

Reply:
That makes sense if we want to converge all our proxies running on the "APIcast framework".

Reply:
Reserved 9421.

Reply:
We should converge them all, now with policies it should be easier ;)